jenkins/linux_install_deps.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Please add this script to the .vsts-ci.yml for some platform(?).

Reply:
That will come afterwards. But it will happen